update dbt cloud data observability documentation (#36538)
DataDog/documentation: content/en/data_observability/jobs_monitoring/dbt.md

2. Click on {{< ui >}}+ Create Service Token{{< /ui >}}.
3. Provide a name for the token.
4. Set the token permissions:
   - If you create the webhook in dbt Cloud yourself, use the {{< ui >}}Stakeholder/Read-Only{{< /ui >}} permission set scoped to the relevant dbt Cloud projects.
   - If Datadog creates and manages the webhook, use {{< ui >}}Developer{{< /ui >}} permissions for the dbt Cloud Enterprise plan or {{< ui >}}Account Admin{{< /ui >}} permissions for the dbt Cloud Team plan.
5. Click {{< ui >}}Save{{< /ui >}} and copy the generated API token.

## Connect your dbt Cloud account to Datadog

Use the API token to configure the integration in Data Observability.

1. Navigate to [{{< ui >}}Datadog Data Observability{{< /ui >}} > {{< ui >}}Settings{{< /ui >}}][1].
2. In the {{< ui >}}dbt Cloud{{< /ui >}} section, click {{< ui >}}Configure{{< /ui >}}.
3. If you have already created a dbt Cloud integration account, make sure you have updated it with the API token with permissions described above.

> 4. If not, create a new account. Fill in the {{< ui >}}Account Name{{< /ui >}}, {{< ui >}}Account Id{{< /ui >}}, {{< ui >}}Account Url{{< /ui >}}, and {{< ui >}}API Token{{< /ui >}} sections.

Reviewer: ```suggestion
4. If not, create an account. Fill in the {{< ui >}}Account Name{{< /ui >}}, {{< ui >}}Account Id{{< /ui >}}, {{< ui >}}Account Url{{< /ui >}}, and {{< ui >}}API Token{{< /ui >}} sections.
```

Author: Fixed.

5. Click {{< ui >}}Save{{< /ui >}} to save your settings.

## Configure webhooks

In Data Observability settings, expand the dbt Cloud account and choose how Datadog receives dbt Cloud job run events.

### Create the webhook in dbt Cloud yourself

Use this option if you want to use a {{< ui >}}Stakeholder/Read-Only{{< /ui >}} service token for artifact ingestion.

1. Select {{< ui >}}I'll manage the webhook in dbt Cloud myself{{< /ui >}}.
2. Copy the Datadog webhook URL.
3. In dbt Cloud, go to {{< ui >}}Account Settings{{< /ui >}} > {{< ui >}}Webhooks{{< /ui >}} > {{< ui >}}Create New Webhook{{< /ui >}}.
4. Paste the Datadog webhook URL into the webhook URL field.
5. Enable the {{< ui >}}Job Run Started{{< /ui >}} and {{< ui >}}Job Run Completed{{< /ui >}} events. To scope ingestion to specific jobs, select those jobs in the dbt Cloud webhook configuration.
6. Save the webhook in dbt Cloud.
7. Copy the HMAC secret from dbt Cloud, paste it into the {{< ui >}}HMAC secret from dbt Cloud{{< /ui >}} field in Datadog, and click {{< ui >}}Save{{< /ui >}}.

> **Note**: Manually created webhooks can take up to 5 minutes after you save the user-managed webhook configuration in Datadog to start accepting traffic from dbt Cloud.

Reviewer: ```suggestion
**Note**: After saving, webhooks you create yourself can take up to 5 minutes to start accepting traffic from dbt Cloud.
```

Author: Fixed.
